pom.xml

```diff
@@ -65,7 +65,7 @@
 		<connection>scm:git:http://gitbox.apache.org/repos/asf/logging-log4cxx.git</connection>
 		<developerConnection>scm:git:https://gitbox.apache.org/repos/asf/logging-log4cxx.git</developerConnection>
 		<url>https://gitbox.apache.org/repos/asf?p=logging-log4cxx.git</url>
-		<tag>HEAD</tag>
+		<tag>v0.11.0-RC1</tag>
```

Comment:
This should be left as `HEAD` as the `mvn release:*` commands will handle setting the value for the appropriate tag commit.

Reply:
Unless this PR has a staged release candidate it seems?

Reply:
What is a staged release candidate?